pkgs/by-name/st/strfry/package.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gcc,
+  perl,
+  openssl,
+  lmdb,
+  flatbuffers,
+  libuv,
+  libnotify,
+  secp256k1,
+  zlib-ng,
+  git,
+  zstd,
+}:
+stdenv.mkDerivation {
+  pname = "strfry";
+  version = "1.0.4";
+  src = fetchFromGitHub {
+    owner = "hoytech";
+    repo = "strfry";
+    tag = "1.0.4";
+    hash = "sha256-2+kPUgyb9ZtC51EK66d3SX2zyqnS6lju2jkIhakcudg=";
+    fetchSubmodules = true;
+    leaveDotGit = true;
```

Comment:
```suggestion
```

Reply:
I encountered some troubles with golpe and thought I needed to preserve .git
I just tried it again and it works without it, so I will remove it